string elevator   # use "up" or "down" 
---
bool success      # task reached
